src/viam/sdk/services/mlmodel.cpp

```diff
@@ -100,6 +100,20 @@ const char* MLModelService::tensor_info::data_type_to_string(const data_types da
     return nullptr;
 }
 
+std::ostream& operator<<(std::ostream& os, MLModelService::tensor_info::data_types dt) {
+    const char* str = MLModelService::tensor_info::data_type_to_string(dt);
+
+    if (str) {
+        os << str;
+    } else {
+        // Cast to unsigned because uint8_t is unsigned char, and 0-9 are whitespace or non printing
+        // characters
+        os << static_cast<unsigned>(static_cast<std::underlying_type_t<decltype(dt)>>(dt));
```

Comment:
Naive question: why did you need to `static_cast` this twice? Couldn't you just cast straight to `unsigned` and be done?